app/controllers/v2/MoveCasesController.scala

```diff
@@ -408,7 +408,7 @@ class MoveCasesController @Inject() (
       .flatten
       .map(c =>
         for {
-          updatedCase <- casesService.updateCase(c.copy(assignee = user, queueId = Some(teamId)))
+          updatedCase <- casesService.updateCase(c.copy(assignee = user, queueId = Some(teamId)), user.getOrElse(Operator(originalUserId)))
```

Comment:
I think this should be `request.operator` as the second argument is the `operatorUpdating`. At the moment it will use the new assignee for the case as the person that did the update I think?

Reply:
Thanks. Resolved